app/controllers/account_controller.rb

```diff
@@ -85,7 +85,7 @@ def lost_password
       return
     else
       if request.post?
-        user = User.find_by_mail(params[:mail])
+        user = User.find_by(mail: params[:mail])
```

Comment:
Note that there is a explicit method `find_by_mail` for users, this is not one of those deprecated dynamic finders.